From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Fri, 11 Apr 2014 09:24:33 +0100
Subject: automake options

Add two automake options.  The first, subdir-objects, is to avoid
zillions of warnings concerning refering to source files in
subdirectories.  The second, -Wall, has no discernible effect, which
is good.
---
 Makefile.am  | 2 --
 configure.ac | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 716009a..ecbebce 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,3 @@
-AUTOMAKE_OPTIONS=	foreign
-
 bin_PROGRAMS=	scrypt
 dist_man_MANS=$(scrypt_man_MANS)
 
diff --git a/configure.ac b/configure.ac
index 93822ad..c2306bd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,6 @@
 AC_INIT([scrypt],[1.1.6])
 AC_CONFIG_AUX_DIR([config.aux])
-AM_INIT_AUTOMAKE()
+AM_INIT_AUTOMAKE([foreign subdir-objects -Wall])
 AM_MAINTAINER_MODE
 AC_PROG_CC
 AC_PROG_RANLIB
